client/src/provides.ts

```diff
@@ -111,13 +114,13 @@ export interface Handler {
   /* Save pending changes to persistence layer */
   save(set?: string): Promise<void>;
   /* Select and seek to track */
-  trackSeek(AnnotationId: AnnotationId): void;
+  trackSeek(AnnotationId: AnnotationId, modifiers?: { ctrl: boolean } | null): void;
   /* Seek Frame */
   seekFrame(frame: number): void;
   /* Toggle editing mode for track */
   trackEdit(AnnotationId: AnnotationId): void;
   /* toggle selection mode for track */
-  trackSelect(AnnotationId: AnnotationId | null, edit: boolean): void;
+  trackSelect(AnnotationId: AnnotationId | null, edit: boolean, modifiers: { ctrl: boolean } | null): void;
```

Comment:
was thehere a reason to make modifiers `null` instead of being an optional undefined if it isn't provided?
I know we didn't do it with edit and we probably should have, but I think modifiers is something we can be an optional object.

Reply:
Fixed by 27d24098